src/client/Components/Routes.tsx

```diff
@@ -1,22 +1,44 @@
 import React from "react";
-import { IonPhaser } from "@ion-phaser/react";
+import { Action } from "redux";
 import { connect } from "react-redux";
-import { HashRouter as Router, Route } from "react-router-dom";
-import PropTypes from "prop-types";
+import { ThunkDispatch } from "redux-thunk";
+import { IRootState as AppState } from "../store/Reducers";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { getLoginStatus } from "../store/Reducers/loginReducer/asyncActions";
```

Comment:
why import from reducers? you should export the reducer to store.js, then just import everything from the store